tests/sentry/workflow_engine/endpoints/test_organization_detector_details.py

```diff
@@ -224,6 +226,7 @@ def test_update(self) -> None:
         query_subscription = QuerySubscription.objects.get(id=data_source.source_id)
         snuba_query = SnubaQuery.objects.get(id=query_subscription.snuba_query.id)
         self.assert_snuba_query_updated(snuba_query)
+        mock_schedule_update_project_config.assert_called_once_with(detector)
```

Comment:
There are no tests for this today so I'm just making sure it's called 🤷‍♀️ the actual behavior is a little beyond my team's scope / responsibility